src/Table/Table.jsx

```diff
@@ -5,17 +5,37 @@ import React, {Component, PropTypes} from "react";
 import * as tablePropTypes from "./tablePropTypes";
 import Cell from "./Cell";
 import Column from "./Column";
+import Footer from "./Footer";
 import Header from "./Header";
 import sortDirection from "./sortDirection";
 
 require("./Table.less");
 
+const DEFAULT_PAGE_SIZE = 10;
+
 
 export class Table extends Component {
   constructor(props) {
     super(props);
 
-    this.state = {};
+    this.state = {
+      currentPage: props.initialPage || 1,
+    };
+  }
+
+  /**
+   * @param {number} page - 1-based index of the page to select.
```

Comment:
thought it would be confusing if the page number is out of sync with what's displayed. in either case, we'll have to deal with a disconnect. no strong feelings on which way that disconnect happens